catroid/src/main/java/org/catrobat/catroid/io/DeviceUserDataAccessor.java

```diff
@@ -83,20 +83,24 @@ public boolean readUserData(UserData userData) {
 		return true;
 	}
 
-	public void writeUserData(UserData userData) throws IOException {
+	public void writeUserData(UserData userData) {
 		synchronized (getLock()) {
-			if (!deviceFile.exists() && !deviceFile.createNewFile()) {
```

Comment:
Why did you delete this method call? It is necessary here

Reply:
so that try and catch blocks could be removed from the places using this method, as you suggested, if this method would throw IO Exception, then try and catch blocks would be necessary, if you want I can change it back : )

Reply:
I also think this would be necessary here, so shall I change it back?